src/measure_theory/group/measure.lean

```diff
@@ -103,6 +104,18 @@ lemma measure_preserving.mul_right (μ : measure G) [is_mul_right_invariant μ]
   measure_preserving (λ x, f x * g) μ' μ :=
 (measure_preserving_mul_right μ g).comp hf
 
+@[to_additive]
```

Comment:
This will probably clash with #17590...

Reply:
Oh indeed, what a striking coincidence. I've added #17590 as a dependency to this PR.